resources/migrations/5_v0.5.14.up.sql

```diff
@@ -0,0 +1,129 @@
+--aws_apigatewayv2_vpc_links
```

Comment:
always ALTER TABLE IF EXISTS to avoid any errors if the table doesn't exist we don't need to do changes.